device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/fileupload/FileUpload.java

```diff
@@ -116,10 +116,8 @@ public synchronized void uploadToBlobAsync(
 
     /**
      * Close the file upload cancelling all existing uploads and shutting down the thread pool.
-     *
-     * @throws IOException if an I/O error occurs in the inputStream.
      */
-    public void closeNow() throws IOException
+    public void closeNow()
```

Comment:
public API?

Reply:
Nope. We construct this internally, but we never ask users to create an instance of this class